dags/utils/onepassword.py

```diff
@@ -0,0 +1,31 @@
+import os
+
+from onepasswordconnectsdk.client import Client, new_client
+import onepasswordconnectsdk
+
+ONEPASSWORD_CONNECT_HOST = os.getenv("OP_CONNECT")
+ONEPASSWORD_CONNECT_TOKEN = os.getenv("OP_API_TOKEN")
+VAULT_ID = os.getenv("OP_VAULT_ID")
+
+client: Client = new_client(ONEPASSWORD_CONNECT_HOST, ONEPASSWORD_CONNECT_TOKEN)
+
+
+def load_dict(required_secrets):
+    """Return dict of specified keys and secrets from 1Password vault.
+
+    Keyword arguments:
+    required_secrets -- a dict of specified keys and values of location config w/o vault id
+    """
+    for value in required_secrets.values():
+        value["opvault"] = VAULT_ID
+
+    return onepasswordconnectsdk.load_dict(client, REQUIRED_SECRETS)
```

Comment:
I think this line needs to be changed to reference the lower-case variable name `requiered_secrets`.

Reply:
you are a lifesaver 🛟 - yep, this was a last minute change. Just fixed it. 🙏